source/reference/limitations.txt

```diff
@@ -0,0 +1,73 @@
+.. _c2c-limitations:
+
+===========
+Limitations
+===========
+
+.. include:: /includes/preview-warning
+
+{+c2c-full-product-name+} is under active development. This preview
+release has several limitations.
+
+General Limitations
+-------------------
+
+- The minimum supported version is MongoDB 6.0.
+- The minimum support Feature Compatibility Version is 6.0. 
+- MongoDB Community Edition is not supported.
+- The source and destination clusters must have the same release
+  version.
+- Environmental assumptions are not validated as pre-checks. 
+- The Primary cluster cannot be changed while syncing.
+- If write blocking is disabled,the client must prevent writes to the
+  source cluster after starting the commit process.
+- DDL operations are not supported during syncing. 
+- Filtered synchronization is not supported.
+- Network compression is not supported.
+- applyOps operations from the source cluster are not supported.
+
+Unsupported Collection Types
+----------------------------
+
+- Collections with collation are not supported.
+- Capped collections are not supported.
+- Time-series collections are not supported.
+
+
+Sharded Clusters
+----------------
+
+- The shard topologies must be the same in the source and target
+  clusters. The following configurations are not supported.
+
+  - Replica set to sharded cluster.
+  - Sharded cluster to replica set.
+  - M to N number of shards.
+
+- There is no replication for zone configuration.
+- Shards cannot be added or deleted while synchronizing.
+- Only indexes which exist on all shards are synchronized.
+- The shard key cannot be redefined RefineShardKey during resharding.
+
+
+Reversing
+---------
+
+- If the old source has unique indexes which ar partially distributed
+  across shards, reversing may cause CEA failures.
```

Comment:
typo

what are CEA failures?

Reply:
CEA: change event application
This is a result of the design decision we made to "Only replicate indexes exist on all shards". `mongosync` doesn't replicate [inconsistent Indexes across shards](https://www.mongodb.com/docs/v5.0/tutorial/manage-indexes/#find-inconsistent-indexes-across-shards). As a result, `mongosync` won't try to relax the constraints for inconsistent unique indexes across shard, and so the existence of an uniqueness index in some but not all shards may cause change event application to fail. I think maybe we could somehow link these two limitations together.

Reply:
We should not say "CEA" in the docs, we don't explain this term anywhere.

Reply:
Done